release-notes/6.0/6.0.0.md

```diff
@@ -0,0 +1,228 @@
+# Release Notes
+
+## [Stable release 6.0.0] - 2024-12-09
+
+This update brings the below changes over the previous stable release:
+
+### Breaking Changes
+
+- Removed support for .NET Standard. [#2386](https://github.com/dotnet/SqlClient/pull/2386)
+- Removed support for .NET 6 [#2927](https://github.com/dotnet/SqlClient/pull/2927)
+- Removed UWP (uap) references. [#2483](https://github.com/dotnet/SqlClient/pull/2483)
+- Removed SQL 2000 client-side debugging support for .NET Framework [#2981](https://github.com/dotnet/SqlClient/pull/2981), [#2940](https://github.com/dotnet/SqlClient/pull/2940)
+
+### Added
+
+#### JSON Support [#2916](https://github.com/dotnet/SqlClient/pull/2916), [#2892](https://github.com/dotnet/SqlClient/pull/2892), [#2891](https://github.com/dotnet/SqlClient/pull/2891), [#2880](https://github.com/dotnet/SqlClient/pull/2880), [#2882](https://github.com/dotnet/SqlClient/pull/2882), [#2829](https://github.com/dotnet/SqlClient/pull/2829), [#2830](https://github.com/dotnet/SqlClient/pull/2830)
```

Comment:
I would suggest we retain these features with their PR reference in list form along with other additions, before providing detailed overview..

The links to PRs makes the heading more complex to look at and lack of full list of changes in list form is not ideal for anyone landing on this page. You will also need the list for GitHub Release publishing as there we do not publish feature details in depth.

Reply:
Ok, I'll split the difference here. I removed the PR numbers from the header, but I don't want to add the items to the list of msc additions. Having the additions called out specially and then also included in the catch-all bucket of msc additions doesn't make a ton of sense. In this way, the important features are called out, the PR numbers are available, and they don't clutter up the header. 

It's fine if we don't publish the feature details in the github release, we just put an entry in the release for the called out items. 30s tops. Besides, I'm gonna be the one authoring it anyhow ...